src/core/src/test/kotlin/org/apache/jmeter/util/CalculatorTest.kt

```diff
@@ -66,11 +66,11 @@ class CalculatorTest {
     @Test
     fun mean() {
         assertEquals(0.0, calculator.mean, "mean()")
-        calculator.addSample(SampleResult(10, 40))
+        calculator.addSample(SampleResult(10 * 1000000L, 40 * 1000000L))
         assertEquals(40.0, calculator.mean, 0.001, "mean(40)")
-        calculator.addSample(SampleResult(10, 42))
+        calculator.addSample(SampleResult(10 * 1000000L, 42 * 1000000L))
         assertEquals((40.0 + 42.0) / 2, calculator.mean, 0.001, "mean(40, 42)")
-        calculator.addSample(SampleResult(10, 48).apply { sampleCount = 2 })
+        calculator.addSample(SampleResult(10 * 1000000L, 48 * 1000000L).apply { sampleCount = 2 })
```

Comment:
It does not sound right to modify the existing tests. Please refrain from modifying the existing tests unless it is absolutely necessary.
Otherwise it looks like you break backward compatibilty as we can't require all the plugins to rewrite and recompile their code to add `* 1000000L` all over the place.